pkg/cmd/server/bootstrappolicy/controller_policy.go

```diff
@@ -218,6 +218,24 @@ func init() {
 			eventsRule(),
 		},
 	})
+
+	// resource-quota-manager
+	addControllerRole(rbac.ClusterRole{
+		ObjectMeta: metav1.ObjectMeta{Name: saRolePrefix + InfraResourceQuotaControllerServiceAccountName},
+		Rules: []rbac.PolicyRule{
+			// TODO: Add rules
```

Comment:
> @enj ideally I would like to merge upstream rules with origin

We aren't going to modify any upstream rules.  If more permissions are required, we'll do that with a delta role in openshift and a separate binding.